src/test/java/uk/gov/justice/laa/crime/cfecrime/CfeControllerTest.java

```diff
@@ -82,4 +82,22 @@ void invalidJsonProducesErrorResult() throws Exception {
 
         assertEquals(HttpStatus.BAD_REQUEST.value(), response.getStatus());
     }
+
+    @Test
+    void exceptionJsonProducesErrorResult() throws Exception {
+        CfeCrimeRequest cfeCrimeRequest = new CfeCrimeRequest();
+        RequestTestUtil.setAssessment(cfeCrimeRequest);
+        RequestTestUtil.setSectionInitMeansTest(cfeCrimeRequest, CaseType.INDICTABLE, MagCourtOutcome.COMMITTED_FOR_TRIAL);
+
+        var content = RequestTestUtil.getRequestAsJson(cfeCrimeRequest);
+
+        MockHttpServletResponse response = mvc.perform(
+                        post("/v1/assessment")
+                                .accept(MediaType.APPLICATION_JSON)
+                                .contentType(MediaType.APPLICATION_JSON)
+                                .content(content))
+                .andReturn().getResponse();
+
+        assertEquals(HttpStatus.BAD_REQUEST.value(), response.getStatus());
```

Comment:
It would be good to check the body also. Presumably we're returning JSON in this case?